data/formats-data.js

```diff
@@ -828,7 +828,7 @@ exports.BattleFormatsData = {
 	},
 	golemalola: {
 		randomBattleMoves: ["stealthrock", "stoneedge", "thunderpunch", "earthquake", "toxic"],
-		randomDoubleBattleMoves: ["doubleedge", "stoneedge", "rockslide", "protect"],
+		randomDoubleBattleMoves: ["doubleedge", "stoneedge", "rockslide", "thunderpunch", "earthquake", "protect"],
```

Comment:
Has this been rolling without Galvanize? Because looking at the scripts file, that's a bigger issue and Thunderpunch isn't needed if it can rely on Double Edge for the Electric STAB.